docs/kyma/docs/037-gs-installation-overrides.md

```diff
@@ -0,0 +1,184 @@
+
+```
+spec:
+  containers:
+  - name: {{ .Chart.Name }}
+	args:
+	  - "/connectorservice"
+	  - '--appName={{ .Chart.Name }}'
+	  - "--domainName={{ .Values.global.domainName }}"
+	  - "--tokenExpirationMinutes={{ .Values.deployment.args.tokenExpirationMinutes }}"
+```
+
+The following fragment of the `values.yaml` file in `connector-service` chart defines the default value for `tokenExpirationMinutes`:
+
+```
+deployment:
+  args:
+    tokenExpirationMinutes: 60
+```
+
+To override this value, such as to change "60 to "90", do the following:
+
+- Create a ConfigMap in the `kyma-installer` Namespace labeled with `installer:overrides` or reuse existing one.
+- Add an entry `application-connector.connector-service.deployment.args.tokenExpirationMinutes: 90` to the map.
+
+Notice that the user-provided override key now contains two parts:
+
+- The chart "path" inside the top-level `core` chart: `application-connector.connector-service`
+- The original template value reference from the chart without the .Values. prefix: `deployment.args.tokenExpirationMinutes`.
+
+Once the installation starts, the Installer generates overrides based on the map entries. The system uses the value of "90" instead of the default value of "60" from the `values.yaml` chart file.
+
+
+#### Global overrides
+
+There are several important parameters usually shared across the charts.
+Helm convention to provide these requires the use of the `global` override key.
+For example, to define the `global.domain` override, just use "global.domain" as the name of the key in ConfigMap or Secret for the Installer.
+
+Once the installation starts, the Installer merges all of the map entries and collects all of the global entries under the `global` top-level key to use for installation.
+
+
+## Merging and conflicting entries
+
+When the Installer encounters two overrides with the same key prefix, it tries to merge them.
+If both of them represent a map (they have nested sub-keys), their nested keys are recursively merged.
+If at least one of keys points to a final value, the Installer performs the merge in a non-deterministic order, so either one of the overrides is rendered in the final YAML data.
+
+It is important to avoid overrides having the same keys for final values.
+
+
+### Example of non-conflicting merge:
+
+Two overrides with a common *key prefix* ("a.b"):
+
+```
+"a.b.c": "first"
+"a.b.d": "second"
+```
+
+The Installer yields correct output:
+
+```
+a:
+  b:
+    c: first
+    d: second
+```
+
+### Example of conflicting merge:
+
+Two overrides with *the same* key ("a.b"):
+
+```
+"a.b": "first"
+"a.b": "second"
+```
+
+The Installer yields either:
+
+```
+a:
+  b: "first"
+```
+
+Or (due to non-deterministic merge order):
+
+```
+a:
+  b: "second"
+```
+
+
+## Values and types
+
+Installer generally recognizes all override values as strings. It internally renders overrides to Helm as a YAML stream with only string values.
```

Comment:
I've written "YAML" on purpose (instead of "yaml") since this is the name of the data format and that's the convention used in technical documents (JSON is another example).

Reply:
ok